doc/src/properties/frame.toml

```diff
@@ -36,15 +36,23 @@ CML = "The text of the ``title`` attribute in a molecule node. Only set if the a
 [md_step]
 type = "number"
 
+TRR = "Actual step of the molecular dynamics simulation."
 XTC = "Actual step of the molecular dynamics simulation."
 
 [time]
 type = "number"
 
+TRR = "The time of the frame in pico seconds."
 XTC = "The time of the frame in pico seconds."
 
 [xtc_precision]
 type = "number"
 
 XTC = """The precision used to compress the coordinates. Only used for ten or more atoms.
 Default is 1000."""
+
+[has_positions]
+type = "bool"
+
+TRR = """Set to ``true`` if the frame contains positions. All positions are zero if not.
+This is a workaround, because chemfiles does not support optional positions yet."""
```

Comment:
I would remove the "This is a workaround, because chemfiles does not support optional positions yet.", since it could be interpreted as "We are going to support optional positions in the future", which is not certain =)

Reply:
Sure, it wasn't my intention to signal changes. Just thought it might be helpful to explain why the interface is different between velocities and positions :)